github/repos_pages.go

```diff
@@ -43,10 +44,17 @@ type PagesBuild struct {
 	UpdatedAt *Timestamp  `json:"updated_at,omitempty"`
 }
 
+// PageUpdateOptions represents options for updating a GitHub Page.
+type PageUpdate struct {
+	CName  *string `json:"cname,omitempty"`
+	Source *string `json:"source,omitempty"`
+}
+
 // EnablePages enables GitHub Pages for the named repo.
 //
 // GitHub API docs: https://developer.github.com/v3/repos/pages/#enable-a-pages-site
 func (s *RepositoriesService) EnablePages(ctx context.Context, owner, repo string) (*Pages, *Response, error) {
+
```

Comment:
Please remove this blank link to be consistent with Go style and the rest of this repo.